client/app/src/accounts/account-transactions.controller.js

```diff
@@ -15,27 +15,28 @@
     const vm = this
 
     vm.INCREASE_API = 50
-    vm.INCREASE_SCROLL = 10
+    vm.INCREASE_SCROLL = 25
 
     vm.address = undefined
     vm.transactions = []
 
-    vm.pageSize = 10
+    vm.pageSize = 25
```

Comment:
Would probably be better to replace this with a constant. Either by creating a new variable or by assigning the value from vm.INCREASE_SCROLL (if the two will always be equal pageSize <-> INCREASE_SCROLL).

Reply:
Well it was coded that way by Lucio, if you want change it.